lib/utils.pm

```diff
@@ -460,6 +476,18 @@ sub reboot_gnome {
     workaround_type_encrypted_passphrase;
 }
 
+=head2 reboot
+
+    reboot;
+
+Reboot system from root console.
+=cut
+sub reboot {
+    select_console 'root-console';
+    type_string "reboot\n";
+    xen_restore_system;
```

Comment:
can you please duplicate if(check_var('VIRSH_VMM_FAMILY', 'xen'))  - just to improve code readability